suse_openstack_cloud

```diff
@@ -396,13 +396,12 @@ fi
 section_header "RabbitMQ broker"
 
 if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+    rabbitmq_var_log="/var/log/rabbitmq"
+    rabbitmq_report_log=$rabbitmq_var_log/rabbitmq_report.log
     source /etc/rabbitmq/rabbitmq-env.conf
-    pushd `pwd`
     cd /var/lib/rabbitmq
```

Comment:
This is because `/usr/lib64/rabbitmq/bin/rabbitmqctl` can be run only from this directory, I suppose that it is because `.erlang.cookie` file. Even in `/usr/sbin/rabbitmqctl` which is bash script there is `cd /var/lib/rabbitmq` command on the beginning